daprdocs/content/en/developing-applications/building-blocks/bindings/howto-triggers.md

```diff
@@ -18,79 +18,271 @@ Dapr bindings allow you to:
+```
+
+{{% /codetab %}}
+
+{{% codetab %}}
+
+```python
+#dependencies
+import logging
+from dapr.ext.grpc import App, BindingRequest
+
+#code
+app = App()
+
+@app.binding('checkout')
+def getCheckout(request: BindingRequest):
+    logging.basicConfig(level = logging.INFO)
+    logging.info('Received Message : ' + request.text())
+
+app.run(6002)
+
+```
+
+Navigate to the directory containing the above code, then run the following command to launch a Dapr sidecar and run the application:
+
+```bash
+dapr run --app-id checkout --app-port 6002 --dapr-http-port 3602 --app-protocol grpc -- python3 CheckoutService.py
+```
+
+{{% /codetab %}}
+
+{{% codetab %}}
+
+```go
+//dependencies
+import (
+	"encoding/json"
+	"log"
+	"net/http"
+	"github.com/gorilla/mux"
+)
+
+//code
+func getCheckout(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	var orderId int
+	err := json.NewDecoder(r.Body).Decode(&orderId)
+	log.Println("Received Message: ", orderId)
+	if err != nil {
+		log.Printf("error parsing checkout input binding payload: %s", err)
+		w.WriteHeader(http.StatusOK)
+		return
+	}
+}
+
+func main() {
+	r := mux.NewRouter()
+	r.HandleFunc("/checkout", getCheckout).Methods("POST", "OPTIONS")
+	http.ListenAndServe(":6002", r)
+}
+
+```
+
+Navigate to the directory containing the above code, then run the following command to launch a Dapr sidecar and run the application:
+
+```bash
+dapr run --app-id checkout --app-port 6002 --dapr-http-port 3602 --dapr-grpc-port 60002 go run CheckoutService.go
+```
+
+{{% /codetab %}}
+
+{{% codetab %}}
 
 ```javascript
-res.status(500).send()
+//dependencies 
+import { DaprServer, CommunicationProtocolEnum } from 'dapr-client'; 
+
+//code
+const daprHost = "127.0.0.1"; 
+const serverHost = "127.0.0.1";
+const serverPort = "6002"; 
+const daprPort = "3602"; 
+
+start().catch((e) => {
+    console.error(e);
+    process.exit(1);
+});
+
+async function start() {
+    const server = new DaprServer(serverHost, serverPort, daprHost, daprPort, CommunicationProtocolEnum.HTTP);
+    await server.binding.receive('checkout', async (orderId) => console.log(`Received Message: ${JSON.stringify(orderId)}`));
+    await server.startServer();
+}
+
 ```
 
+Navigate to the directory containing the above code, then run the following command to launch a Dapr sidecar and run the application:
+
+```bash
+dapr run --app-id checkout --app-port 6002 --dapr-http-port 3602 --dapr-grpc-port 60002 dotnet npm start
+```
+
+{{% /codetab %}}
+
+{{< /tabs >}}
+
+### ACK-ing an event
+
+In order to tell Dapr that you successfully processed an event in your application, return a `200 OK` response from your HTTP handler.
+
+### Rejecting an event
+
+In order to tell Dapr that the event was not processed correctly in your application and schedule it for redelivery and handle it gracefully using logs or exceptions.
```

Comment:
Lets put this back to how it was until we discuss how best this should be handled. 

```suggestion
In order to tell Dapr that the event wasn't processed correctly in your application and schedule it for redelivery, return any response different from `200 OK`. For example, a `500 Error`.
```